sympy/physics/mechanics/tests/test_eombase.py

```diff
@@ -0,0 +1,30 @@
+import sympy.physics.mechanics.eombase as eombase
+
+# Determine what inputs should be used
+EOM = eombase.EOMBase(frame=None)
+
+# Should contain access to all the property attributes contained in existing EOM
+# generators
+
+Thing = "Place holder until a correct output is chosen"
+
+assert EOM.auxiliary_eqs == Thing
```

Comment:
This is only relevant to Kane's method, I think. Probably shouldn't be in the base class.